src/chttpd/src/chttpd_db.erl

```diff
@@ -1453,6 +1525,18 @@ parse_engine_opt(Req) ->
             end
     end.
 
+
+parse_partitioned_opt(Req) ->
+    case chttpd:qs_value(Req, "partitioned") of
```

Comment:
Can you add a `"false"` case here so we can support `PUT /db?partitioned=false` requests?
Also, it'd be nice to normalise the case using `string:to_lower` (or similar) like we do for other parameters.

Reply:
we now handle false but I don't think there's much precedence for case-insensitivity